_posts/2021-03-04-grafana.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Grafana and Loki demo"
+date: 2021-03-04 09:30
+categories: meetings
```

Comment:
I think this should be `upcoming` so it appears in the right hand "Upcoming meetings" column of the page